lib/SdkConfig.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Global configuration of the SDK.
+ */
+export default class SdkConfig {
+  /**
+   * Default hash algorithm used when hashing is performed.
+   */
+  public static hashAlgorithmInMultihashCode = 18; // SHA256
+
+  /**
+   * Maximum bytes for canonicalized delta.
+   */
+  public static maxCanonicalizedDeltaSizeInBytes: number = 1000;
```

Comment:
I think this should be 2000. I changed it to 2000 in https://github.com/decentralized-identity/sidetree/pull/866/files.

(Definitely not secretly sneaking in a pr for you to review :p)

Reply:
Wait, why is it not 1000?